src/components/inject.ts

```diff
@@ -29,3 +29,57 @@ export function inject<T>(element: HTMLElement, key: symbol): T | undefined {
   }
   return undefined
 }
+
+/**
```

Comment:
simplify:
* drop all comments
* simplify signature to: export function unprovide(element: HTMLElement): void